collections.md

```diff
@@ -1158,7 +1158,7 @@ The `isNotEmpty` method returns `true` if the collection is not empty; otherwise
 The `join` method joins the collection's values with a string. Using this method's second argument, you may also specify how the final element should be appended to the string:
 
     collect(['a', 'b', 'c'])->join(', '); // 'a, b, c'
-    collect(['a', 'b', 'c'])->join(', ', ', and '); // 'a, b, and c'
+    collect(['a', 'b', 'c'])->join(', ', ', and '); // 'a, b and c'
```

Comment:
In fact, there is no problem with the original content. 

more see： https://www.grammarly.com/blog/comma/#:~:text=Julie%20loves%20ice%20cream%2C%20books%2C%20and%20kittens.